### DISCUSSION TITLE
recipes/cookies-and-bars/the-new-york-times-chocolate-chip-cookies/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#961] [Comments]
# recipes/cookies-and-bars/the-new-york-times-chocolate-chip-cookies/

A collection of my recipes.

https://nicholaswilde.io/recipes/cookies-and-bars/the-new-york-times-chocolate-chip-cookies/

**Comment by nicholaswilde** [comment]
Remove
